File: iteration2/CSE3063F21P1_GRP16/src/main/java/Models/E0Success.java
package Models;

import java.util.ArrayList;

public class E0Success implements CourseEvents {
	
	
private ArrayList<Student> students;


public ArrayList<Student> getStudents() {
	return students;
}

public  E0Success() {
	students=new ArrayList<>();
	
}

@Override
public String toString() {

	String s="    Total Number of Student who successfully enrolled to the course: "+students.size();
	if(students.size()>0)
		s+="\n     -List Of Students:\n";
	else
		s+="\n";

	for(int i=0;i<students.size();i++) {
		s+="      *"+students.get(i).getFirstName()+" "+students.get(i).getLastName()+"\n";


	}
	return s;
}

}

File: iteration3/CSE3063F21P1_GRP16/models/E0Success.py
from typing import List
from CourseEvents import CourseEvents
from Student import Student


class E0Success(CourseEvents):
    def __init__(self):
        self._students = []
        
    def getStudents(self):
        return self._students
    
    def toString(self):
        s = "    Total Number of Student who successfully enrolled to the course: {}\n".format(len(self._students))
        if self._students:
            s += "     -List Of Students:\n"
            for i in range(len(self._students)):
                s += "      *{} {}\n".format(self._students[i].getFirstName(), self._students[i].getLastName())
        return s